charts/deploy.just

```diff
@@ -44,7 +44,8 @@ deploy-cluster namespace=defaultNamespace enableHubble=hubbleEnabled:
   helm install cilium cilium/cilium --version 1.15.5 \
       -f ./dev/values/cilium.yml \
       {{ if enableHubble == "true" { '--set hubble.enabled=true --set hubble.relay.enabled=true --set hubble.ui.enabled=true --set hubble.ui.ingress.enabled=true' } else { '' } }} \
-      --namespace kube-system
+      --namespace kube-system \
+      --labels core=true
```

Comment:
adding the label makes it possible to add a command which just deletes all the helm pieces without having to start a new cluster from scratch